core/controllers/base_test.py

```diff
@@ -526,6 +526,11 @@ def test_redirect_oppia_test_server(self):
         self.assertEqual(
             response.headers['Location'], 'https://oppiatestserver.appspot.com')
 
+    def test_splash_redirect(self):
+        # Tests that the old '/splash' URL is redirected to '/'.
+        response = self.get_html_response('/splash', expected_status_int=302)
+        self.assertNotIn('splash', response.headers['location'])
```

Comment:
Not enough. Always assert an equality, where possible.

Reply:
Done!